crates/egui/src/frame_state.rs

```diff
@@ -153,7 +153,7 @@ impl FrameState {
         *unused_rect = screen_rect;
         *used_by_panels = Rect::NOTHING;
         *scroll_target = [None, None];
-        *scroll_delta = Vec2::default();
+        *scroll_delta = (Vec2::default(), style::ScrollAnimation::none());
```

Comment:
```suggestion
        *scroll_delta = Default::default();
```

Reply:
Hmm, I explicitly set it to none since none is not the default. But I think it shouldn't matter since the animation isn't applied unless the Vec2 is not zero, so I'll update it.